pkg/backend/display/object_diff.go

```diff
@@ -220,29 +220,57 @@ func PrintObject(
 	b *bytes.Buffer, props resource.PropertyMap, planning bool,
 	indent int, op deploy.StepOp, prefix bool, debug bool) {
 
+	p := propertyPrinter{
+		dest:     b,
+		planning: planning,
+		indent:   indent,
+		op:       op,
+		prefix:   prefix,
+		debug:    debug,
+	}
+	p.printObject(props)
```

Comment:
We could use the latter, but in the case of a struct literal with many fields I prefer the former in order to avoid losing the method call in the visual noise.